forge-gui/res/cardsfolder/upcoming/shipwreck_sentry.txt

```diff
@@ -0,0 +1,8 @@
+Name:Shipwreck Sentry
+ManaCost:1 U
+Types:Creature Human Pirate
+PT:3/3
+K:Defender
+S:Mode$ CanAttackDefender | ValidCard$ Card.Self | IsPresent$ Artifact.ThisTurnEntered+EnteredUnder You | Description$ As long as an artifact entered the battlefield under your control this turn, CARDNAME can attack as though it didn't have defender.
```

Comment:
you need to check the condition with Count property so it looks at the whole turn